code-challenges.test.js

```diff
@@ -11,59 +11,147 @@
 // Run the file with the following command: $ yarn jest
 
 // --------------------INSTRUCTOR EXAMPLE: Create a function that takes in an array of numbers and returns an array with all the numbers multipled by 3.
-
+// describe("hello", () => {
+//
+//
+//     it("returns a string that says hi", () => {
+//
+//
+//         expect(hello()).toEqual("hi")
+//     })
+// })
+//
 // a) Create a test with expect statements for each of the variables provided.
-
-const numbersArray1 = [6, 7, 8, 9, 10]
+// const numbersArray1 = [6, 7, 8, 9, 10]
 // Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+// const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
-
 // b) Create the function that makes the test pass.
 
 
 
-// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisble by three or not.
+// --------------------1) Create a function that takes a number as an argument and decides if the number is evenly divisible by three or not.
+/**Pseudo Code
+ - Our first step will be to define our function giving it a name.
+ - It will take a number as an argument, so we can give a name to the parameter it will take in , in this case num
+ - In order to decide whether the argument is divisible by three or not, we will use a conditional if/else statement
+   to evaluate the condition.
+ - We can use the modulo operator to accomplish this task.
+ - Based on our two desired returns/outputs "divisible by three", or "not divisible by three" we will log out the return with one of these
+   phrases.
+ - Running Jest with the correctly defined Describe, It , and Expect statements satisfies our need to log at the end.
+ - Instead , if our function is correct , we will receive all passing green for our Jest tests.
+ **/
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const num1 = 15
-// Expected output: "15 is divisible by three"
-const num2 = 0
-// Expected output: "0 is divisible by three"
-const num3 = -7
-// Expected output: "-7 is not divisible by three"
+    describe("divisibleByThree", () => {
 
+    it("returns a string whether the argument passed is divisible by 3", () => {
 
-// b) Create the function that makes the test pass.
+        const num1 = 15
+//      Expected output: "15 is divisible by three"
+        const num2 = 0
+//      Expected output: "0 is divisible by three"
+        const num3 = -7
+//      Expected output: "-7 is not divisible by three"
 
+        expect(divisibleByThree(num1)).toEqual(`${num1} is divisible by three`)
```

Comment:
Best practices to get the test suite to test that the output of your function works correctly: hard code the expected output. For instance, 
Change
expect(divisibleByThree(num1)).toEqual(`${num1} is divisible by three`)
to
expect(divisibleByThree(num1)).toEqual("15 is divisible by three")

So you are telling the test when the function is passed the variable num1, you expect to see "15 is divisible by three". Then you create the function using various techniques.